internal/cephes/polevl.go

```diff
@@ -8,8 +8,15 @@
  * Direct inquiries to 30 Frost Street, Cambridge, MA 02140
  */
 
+/* Sources:
```

Comment:
What are these the sources for? Can you move them closer (inside) the actual function?